draft-ietf-teep-architecture.md

```diff
@@ -1329,23 +1329,47 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy. A compromised TAM can install
-arbitrary TAs to, potentially, many devices, which makes TAMs high value
-targets. An advanced, persistent threat actor could potentially compromise
-a TAM without impacting its certificate or raising concern from the TAM's
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
+and damage to user devices that it can managee and the device owners. 
+In one hand, the device information of many devices that the TAM manages and 
+therefore have in its system may be leaked to a bad actor. On the other hand,
+a compromised TAM can install many TAs to launch somee DoS attack to devices,
+for example, by filling up a device's TEE resources reserved for TAs such that 
+other TAs may not get resource to be installed or properly function. It may
+also install malicious TAs to, potentially, many devices, if it can also 
+get or access a Trusted Component signer key that is trusted by the TEEs. 
```

Comment:
This sentence is about a compromised Trusted Component Signer, not a compromised TAM, I think.  And so it doesn't really belong here in my view.

Reply:
It describes that unless the TAM can have a signer key that is trusted it isn't able to install a malicious or abusive TA. This is to describe a condition for this TAM's distribution of abusive TA. So I think it applies here.

Reply:
I don't follow. I agree with your statement in the comment, but I don't see that in the text above.

Reply:
@dthaler How about the following revision?

"also install malicious TAs to potentially many devices under the condition that it can have a Trusted Component signer key that is trusted by the TEEs."